plugins/generator-1.20.1/forge-1.20.1/livingentity.definition.yaml

```diff
@@ -6,16 +6,15 @@ templates:
   - template: livingentity/livingentity_projectile.java.ftl
     name: "@SRCROOT/@BASEPACKAGEPATH/entity/@NAMEEntityProjectile.java"
     condition: hasCustomProjectile()
+  - template: livingentity/entity_spawn_biome_modifier.json.ftl

```

Comment:
Please keep this file intact as previous structuring was ok as generation was split into own category

Reply:
✔️ Fixed?